dracut/31ignition-ostree/ignition-ostree-populate-var.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Ignition OSTree: Populate /var
+ConditionKernelCommandLine=ostree
```

Comment:
This breaks live PXE.

Reply:
Updated to do
```
ConditionKernelCommandLine=|ostree
ConditionPathExists=|/run/ostree-live
```
Not tested, but does that make sense?

Reply:
Yup!